### DISCUSSION TITLE
Using backdrop Feature?

### DISCUSSION THREAD

**Discussion by jasan-s** [pmndrs/drei#665] [Q&A]
The new Backdrop feature doesn't show on Canvas. Possible to provide an example of backdrop use and demo.

**Comment by drcmda** [comment]
it probably shows but it is white, or it is too small for your scene (1/1/1 by default), or occluded. https://codesandbox.io/s/backdrop-and-cables-2ij9u

try this

```jsx
<Backdrop>
  <meshStandardMaterial color="red" />
</Backdrop>
```

position it, set a long enough floor, set the material.
